astroquery/svo_fps/core.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import requests
+import io
+
+from astropy import units as u
+from astropy.io.votable import parse_single_table
+# The VOTables fetched from SVO contain only single table element, thus parse_single_table
+
+from ..query import BaseQuery
+
+__all__ = ['SvoFpsClass', 'SvoFps']
+
+FLOAT_MAX = np.finfo(np.float64).max
+
+
+class SvoFpsClass(BaseQuery):
+    SVO_MAIN_URL = 'http://svo2.cab.inta-csic.es/theory/fps/fps.php'
+
+    def data_from_svo(self, query, cache=True,
+                      error_msg='No data found for requested query'):
+        """Get data in response to the query send to SVO FPS
+
+        Parameters
+        ----------
+        query : dict
+            Used to create a HTTP query string i.e. send to SVO FPS to get data.
+            In dictionary, specify keys as search parameters (str) and
+            values as required. List of search parameters can be found at
+            http://svo2.cab.inta-csic.es/theory/fps/fps.php?FORMAT=metadata
```

Comment:
This needs clearer description, it is not immediately clear to me how to use the info found at `http://svo2.cab.inta-csic.es/theory/fps/fps.php?FORMAT=metadata` to build a query dict. And example that includes both a min/max type parameter and a option parameter would be good.

Reply:
Punting on this - we should have a method to get and share all those parameters, but:
https://github.com/astropy/astropy/issues/10572